### PR TITLE
5 things icon

### PR DESCRIPTION
**Pull request by burkeholland**
Adds 5 Things icon to the media folder

### CODE REVIEW

**Comment by burkeholland** [comment]
Added section to community page and squashed

**Comment by thedanfernandez** [comment]
@burkeholland - I filed a ticket on this as it looks like validation is stuck, link is here https://mseng.visualstudio.com/CSI/_workitems?id=1177742

**Comment by craigshoemaker** [comment]
@rloutlaw can this be be merged, or do we need to wait for the bug fix? I have had this same issue in a number of my PRs in this repo

File: .gitignore [hunk 1]
```diff
@@ -6,5 +6,6 @@ _themes*/
 node_modules/
 src/
 _dependentPackages/
+.DS_Store
```
Comment (dend) [inline .gitignore]: @burkeholland can you please remove `.DS_Store`?

**Comment by dend** [comment]
@rloutlaw @craigshoemaker this should be OK to merge.

cc: @burkeholland @thedanfernandez

**Pull request merged**